components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -184,13 +187,8 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
             },
             style
         );
-
-    return (
-        <ListItem style={getWrapperStyle()} onClick={(): void => onClick()} dense={dense}>
-            <div className={classes.statusStripe} style={{ backgroundColor: statusColor }} />
-            {divider && (
-                <Divider className={classes.divider} style={{ zIndex: 0, left: divider === 'full' ? 0 : 72 }} />
-            )}
+    const listItemChild = (
```

Comment:
I think this would be more readable inline in the ListItem below instead of broken out as a separate constant unless we're conditionally not showing it or something similar.